buildpipeline/DotNet-CoreFx-Trusted-Windows-NoTest.json

```diff
@@ -62,13 +62,14 @@
       "alwaysRun": false,
       "displayName": "Install Signing Plugin",
       "timeoutInMinutes": 0,
+      "condition": "ne(variables['PB_SignType'], 'oss')",
```

Comment:
This does nothing currently but will when PipeBuild updates to a newer API version.

Reply:
I assume this is a new feature in PipeBuild? and that we will strip this before sending it to VSTS?

Reply:
it's a new feature in VSTS that gets stripped because of the API version we use.  Theoretically (I haven't personally tested this yet) we can now add args to Pipebuild to use API version 3.2, then this syntax will work.

Since the 1.1 branch does not use Pipebuild, I'm putting the condition there to make sure it works as expected.